account_bank_statement_reopen_skip_undo_reconciliation/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 ForgeFlow S.L.
+# @author Jordi Ballester <jordi.ballester@forgeflow.com.com>
+# Copyright 2022 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Bank Statement Reopen Skip Undo Reconciliation",
+    "summary": "When reopening a bank statement it will respect the "
+    "reconciled entries.",
+    "version": "14.0.1.0.0",
+    "author": "ForgeFlow, Akretion, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```
also above copyrights...